src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/lrn.cpp

```diff
@@ -59,6 +59,9 @@ class LRNLayerCPUTest : public testing::WithParamInterface<LRNParams>, public ov
         auto axesNode = ov::op::v0::Constant::create(ov::element::i32, { axes.size() }, axes);
         auto lrn = std::make_shared<ov::op::v0::LRN>(params[0], axesNode, alpha, beta, bias, size);
         function = makeNgraphFunction(inputPrecision, params, lrn, "LRN");
+        if (inputPrecision == ov::element::f32) {
+            abs_threshold = 0.05;
+        }
```

Comment:
The threshold seems to be too high. So as the rule of thumb, abs_threshold > 1e-4 for fp32 is little too high. I do agree that in some cases it could be 5e-3, but 5e-2 seems too much.